clean.py

```diff
@@ -0,0 +1,243 @@
+## Import libraries for functions
+import os
+import pathlib
+import earthpy as et
+import pandas as pd
+import geopandas as gpd
+import matplotlib.pyplot as plt
+import rasterstats as rs
+import xarray as xr
+import rioxarray as rxr
+import seaborn as sns
+
+
+
+fig, (ax1, ax2) = plt.subplots(1, 2, figsize=(10, 10))
```

Comment:
This line shouldn't be here (already included in function, and not the right location anyway)